.readthedocs.yml

```diff
@@ -3,10 +3,10 @@ version: 2
 python:
   version: "3.8"
   install:
-      - method: pip
-        path: .
-        extra_requirements:
-        - sphinx
+  - method: pip
```

Comment:
@mmcky should these be indented inside of `install`, like in this example: https://docs.readthedocs.io/en/stable/config-file/v2.html#python-install